src/common/Pagination/styled.js

```diff
@@ -0,0 +1,120 @@
+import styled, { css, keyframes } from "styled-components";
+import { ReactComponent as arrow } from "./arrow.svg";
+
+export const Wrapper = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+`;
+
+const move = keyframes`
+   0% {
+      transform: translateX(0px);
+   }
+   25% {
+    transform: translateX(-4px);
+   }
+   50% {
+    transform: translateX(0px);
+   }
+   75% {
+    transform: translateX(4px);
+   }
+   100% {
+    transform: translateX(0px);
+   }
+`;
+
+export const Button = styled.button`
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    padding: 10px 16px;
+    border-radius: 5px;
+    font-size: 14px;
+    border: none;
+    background: ${({ theme }) => theme.colors.lightBlueButton};
+    margin: 0px 6px;
+
+    @media (min-width: ${({ theme }) => theme.breakpoints.large}) {
+        &:hover{
+            animation: ${move} 1s linear infinite;
+        }
+    }
+    
+    ${({ next }) => next && css`
+        @media (min-width: ${({ theme }) => theme.breakpoints.large}) {
+            &:hover{
+                animation: ${move} 1s linear infinite reverse;
+            }
+        } 
+    `}
```

Comment:
I would add an animation without a media query so that it appears every time, regardless of screen resolution

Reply:
I can remove media query, but when I pressed the button on the touchscreen, the button starts moving, can you check it in your browser,

Reply:
There is no hover on mobile devices! 😅 Leave the media as you have it, but maybe add :active pseudo-class